File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -0,0 +1,932 @@
+---
+title: Presentation Working with Find and Replace | Syncfusion
+description: This section illustrates how to find a particular text and replace it with another text or part of the document
+platform: file-formats
+control: Presentation
+documentation: UG
+keywords: 
+---
+# Working with Find and Replace
+
+.NET PowerPoint library allows you to search a particular text you like to change and replace it with another text in the PowerPoint presentation.
+
+* You can replace either the first occurrence or all the occurrences of the text with another text in all PowerPoint slides and elements such as shape, textbox, table, smartart, etc.
+* You can replace the text in specific slide, notes slide, master slide or layout slide.
+* You can replace the text by matching case, whole word and all occurrences or first occurrence alone.
+* You can also replace the content that spans across several lines.
+
+## Find and replace in PowerPoint presentation
+
+You can find text in a PowerPoint presentation and replace it with other text.
+
+The following code example illustrates how to find all occurences of a particular text and replace it with other text in a PowerPoint presentation.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+	//Finds all the occurrences of a particular text in the PowerPoint presentation
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text as single text part
```

Comment:
Gets the found text as a single text part

Reply:
Modified as you said.